src/odemis/model/_dataflow.py

```diff
@@ -510,42 +559,33 @@ def __del__(self):
 
 
 class SubscribeProxyThread(threading.Thread):
-    def __init__(self, notifier, uri, max_discard, zmq_ctx):
+    def __init__(self, df_proxy, uri, zmq_ctx):
         """
-        notifier (callable): method to call when a new array arrives
+        # notifier (callable): method to call when a new array arrives
```

Comment:
why the `# `?

Reply:
This has to go away. I'll take care of this.